Improwised/frappe
# Group permission in frappe

## Steps to Test the PR

1. Switch to `feat/group_permission` branch and run `bench migrate`.
2. Create 2 projects and 2 User Groups.
3. Now, create user group permissions and assign different user groups to different projects.
4. Log in as a user belonging to a specific user group and verify if the user can see the assigned user group project.
5. Additionally, add or remove users from user groups simultaneously and verify whether users can see projects or not.

Check video for reference

https://drive.google.com/file/d/1JI_jEMAojttl-g3ZfIl-xmvgk8zUXDFW/view
